.github/labeler.yml

```
@@ -0,0 +1,20 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/dask-cuda/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/dask-cuda/labels
+
+Python:
```

This repo has a `python` label, but not a `Python` label.
```suggestion
python:
```